hack/generate-manifest.sh

```diff
@@ -39,6 +39,7 @@ Generate a YAML manifest for Antrea using Kustomize and print it to stdout.
                                       This option will work only for Kind clusters (when using '--kind').
         --coverage                    Generates a manifest which supports measuring code coverage of Antrea binaries.
         --simulator                   Generates a manifest with antrea-agent simulator included
+        --custom-adm-controller       Generates a manifest with custom Antrea admission controller to validate/mutate resources.
```

Comment:
so the manifest needs to be generated with that flag *and* we need to provide the config option? The config options is always available, and there is no indication in the description about the requirements. What will be the behavior if the config option is enabled but the `MutatingWebhookConfiguration` resource was not defined?

Reply:
without this yaml change, the requests will never reach our service. The service handler is behind the config option. i wonder if we can get rid of the config option and only maintain this yaml generation logic .. what do you think?

Reply:
I feel like we may indeed want to remove the configuration option altogether, except if you want to create the `MutatingWebhookConfiguration` resource from the Antrea Controller using the K8s APIs.

My concerns are:
1) if the config option is enabled, but the resource does not exist, nothing would happen and there would be no warning. However, we could always document the config option better and remind the user that resource needs to exist.
2) if the config option is disabled, but the resource does exist, then namespace creation would fail. That would be a bigger issue.

What do you think?

Reply:
for 2. I think you mean config option is disabled, and the resource exists, then NS creations would fail. Yes this is a valid concern.
I agree, we can remove the config option all together.

Reply:
Agreed with what you guys said here. We can remove the config option.